dandiapi/zarr/models.py

```diff
@@ -89,16 +93,6 @@ def s3_path(self, zarr_path: str) -> str:
         """Generate a full S3 object path from a path in this zarr_archive."""
         return zarr_s3_path(str(self.zarr_id), zarr_path)
 
-    def generate_upload_urls(self, path_md5s: list[dict]):
```

Comment:
This was a bad API, as the `dict` in the parameter was tightly coupled to the one and only place it was invoked from a view. I just moved it inline to the view.